LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 - 2024 Neon Inc.
+Copyright (c) 2022 - 2025 Neon Inc.
```

Comment:
I would simplify to this:

```.md
MIT License
Copyright (c) 2022 Neon Inc.
```

Reply:
I believe legally we need to include the full range of modification dates. I'll DM you a Slack link on this.